.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,104 @@
+name: Bug report
+description: Bugs in Rhino
+title: "Rhino Issue: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before** reporting an issue, make sure to read the [documentation](https://picovoice.ai/docs/rhino/) and search [existing issues](https://github.com/Picovoice/rhino/issues).
+  - type: checkboxes
+    id: check
+    attributes:
+      label: Have you checked the docs and existing issues?
+      description: Make sure you have checked all of the below before submitting an issue
+      options:
+        - label: I have read all of the relevant Picovoice Rhino docs
+          required: true
+        - label: I have searched the existing issues for Rhino
+          required: true
+  - type: dropdown
+    id: sdk
+    attributes:
+      label: SDK
+      options:
```

Comment:
Missing `C` (it's not really an SDK, but if someone is using out libs directly, they are probably using C)

Reply:
do all the other animals need C under SDKs as well?